grant_token_request.go

```diff
@@ -21,6 +21,12 @@ type grantTokenBuilder struct {
 	opts *grantTokenOpts
 }
 
+type SpaceId string
+
+type grantTokenSumBuilder grantTokenBuilder
```

Comment:
This seems like we are introducing a new qualifier `SUM` for the VSP stuff. Is this something the team is doing across all SDKs?